mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -390,104 +392,180 @@ public Flags getFlags() {
             NO_COUNT
         }
 
-        /**
-         * Gets the UIDs of recent messages if requested or an empty
-         * {@link List} otherwise.
-         * 
-         * @return the uids flagged RECENT in this mailbox,
-         */
-        List<MessageUid> getRecent();
+        public static MailboxMetaData sensibleInformationFree(MailboxACL resolvedAcl, UidValidity uidValidity, boolean writeable, boolean modSeqPermanent) throws MailboxException {
```

Comment:
What do you mean by `sensibleInformationFree` ?

Reply:
That is just a code move :-)

Will add java doc.